allen-cell-animated/nucmorph-colorizer
# feature: Added metadata tags and icons

Problem
=======
Closes #264 by adding opengraph metadata tags and icons. Heavily referenced a similar PR from @interim17 😊!

*Estimated size: small, 15 minutes*

Solution
========
- Adds metadata for rich embeds.
- Adds a default SVG favicon.
- Adds an image for embedding.

## Type of change
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Open staging and confirm that the favicon appears: https://staging.timelapse.allencell.org/
2. Copy the staging link into an the opengraph tester found here: https://www.opengraph.xyz/
3. The metadata should all appear complete.

Screenshots (optional):
-----------------------
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/546d6350-ef31-433c-99aa-b001cbc7fec1)

![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/3bae6eb7-ac3a-4576-8658-08be6451e447)


Keyfiles (delete if not relevant):
-----------------------
1. main file/entry point
2. other important file

Thanks for contributing!
